lib/fs.js

```diff
@@ -819,7 +821,8 @@ function write(fd, buffer, offsetOrOptions, length, position, callback) {
   fd = getValidatedFd(fd);
 
   let offset = offsetOrOptions;
-  if (isArrayBufferView(buffer)) {
+  const bufferToWrite = isArrayBuffer(buffer) ? new Uint8Array(buffer) : buffer;
```

Comment:
Should this also support `SharedArrayBuffer`?

Reply:
It probably should. I can't think of any unwanted side effects that would arise from including SharedArrayBuffer in this feature.